console/src/components/UIKit/atoms/Button/Button.style.js

```diff
@@ -0,0 +1,62 @@
+import styled, { css } from 'styled-components';
+import {
+  layout,
+  space,
+  color,
+  border,
+  typography,
+  flexbox,
+} from 'styled-system';
+
+export const ButtonStyles = styled.button`
+  cursor: ${({ disabled }) => !disabled && 'pointer'};
+  appearance: button;
+  
+  /* Pseudo classes like hover are not supported by Styled-System  */
+
+  /* Hover effects when button is not disabled */
+
+  ${props => {
+    const { type, theme, disabled } = props;
+
+    // Default ~ If the received button type is out of the theme range then default box-shadow color value will be assigned.
+
+    const boxShadowColor = theme.buttons[type]
+      ? theme.buttons[type].boxShadowColor
+      : theme.buttons.default.boxShadowColor;
+
+    // In case of Secondary Button when it's not disabled
+
+    if (type === 'secondary' && !disabled) {
+      return css`
+        &:hover {
+          box-shadow: ${`0 2px 8px 0 ${boxShadowColor}`};
```

Comment:
?????

Reply:
I'll add a comment for that or move my logic to the React component itself.